server/api/controllers/UserControllers.js

```diff
@@ -1,9 +1,13 @@
 const User = require("../models/User");
 // const jwt = require("jsonwebtoken");
-const { generateToken } = require("../utilities/auth");
+const { generateToken, generateResetToken } = require("../utilities/auth");
 require("dotenv").config();
+const sgMail = require("@sendgrid/mail");
 
-const secret = process.env.SECRET;
+// const secret = process.env.SECRET;
+const sgAPIKey = process.env.SENDGRID_API_KEY;
```

Comment:
This is more of a reminder to myself to add these to my .env and mention them in the README.